Add data for Aria spec

jpmedley opened the pull request:
Rachel Andrew has an [in progress PR](https://github.com/mdn/content/pull/2001) to add the aria mixins to the `Element` and `ElementInternal` interfaces. When these merge, these will need supporting compat data.

This is just a start. I don't currently have the means to discover support in all browsers. Please advise me on how I should procede.

ddbeck commented:
Hi @jpmedley. To make this work, I think a few things need to happen:

- Share how you got the values you have now. Did you test to confirm the version values you reported in the PR as it stands? Did you set the `false` values based on something other than testing?
- In the case of `ElementInternals`, the features are inconsistent with the [`api.ElementInternals`](https://github.com/mdn/browser-compat-data/blob/6cb0b9acead6ca8ea0e986abae98093608636b31/api/ElementInternals.json) data. Either the data here or the data there needs to change. This won't fail the linter yet, but I'd rather not create a problem that we know we'll have to fix later (related: #9044).
- It'd also be nice to link to the spec IDL here, to help the reviewer(s) confirm that this is complete (and for the benefit of the project history).

That said, @foolip might have some interest in testing these as a group.

jpmedley commented:
* Here's where I got the number for Chrome: https://www.chromestatus.com/feature/5761503818940416. Edge would have inherited this when it arrived as version 79. False was maybe not the best choice for the other values. But null is no longer allowed and **I have no means of testing.**
* I've updated ElementInternals to 77.
* This is based on what we implemented in Chrome. If the spec is different, this will change when the implementation catches up. (In rare cases it's the other way around. Here's the current IDL: https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/dom/aria_attributes.idl

Elchi3 commented:
Drive-by comment: The latest spec IDL is here: https://rawgit.com/w3c/aria/master/#ARIAMixin and the name there is `AriaMixin`.

foolip commented:
These members are tested here:
https://mdn-bcd-collector.appspot.com/tests/api/Element
https://mdn-bcd-collector.appspot.com/tests/api/ElementInternals

I'll see if the update-bcd.js script suggests any updates here.

foolip commented:
The collector suggests that all of this was added in Chrome 81 and some in 83. I found this evidence by looking at the history of `third_party/blink/web_tests/virtual/stable/webexposed/global-interface-listing-expected.txt` in Chromium:

Most in 81: https://storage.googleapis.com/chromium-find-releases-static/dbe.html#dbef8d4838807211e2940f0929c8109cdf06b9d5

`ariaDescription` in 82: https://storage.googleapis.com/chromium-find-releases-static/617.html#617cebed098e061d1c66f63937f9e2b02ceea675

`ariaSetSize` in 84: https://storage.googleapis.com/chromium-find-releases-static/62f.html#62fad45eb95f8feaef20cff4a8f71977e350ede8

I also know there's one attribute extra in Chrome not in the spec or vice versa that I've come across, so checking each individually would be a good idea.

foolip commented:
Since there would be extremely many suggestions to enter manually, I've sent https://github.com/mdn/browser-compat-data/pull/9202 to show the changes. The target of that PR is the branch for this PR, so merging it would bring it into this PR.

jpmedley commented:
Changes merged in. PTAL

foolip commented:
I went ahead and renamed the files per https://github.com/mdn/browser-compat-data/pull/9067#issuecomment-781446524

foolip commented on api/_mixins/ARIAMixin__Element.json:
```diff
@@ -0,0 +1,1830 @@
+              "version_added": "83"
+            },
+            "edge": {
+              "version_added": "81"
```
```suggestion
              "version_added": "83"
```

foolip commented on api/_mixins/ARIAMixin__Element.json:
```diff
@@ -0,0 +1,1830 @@
+              "version_added": "84"
+            },
+            "edge": {
+              "version_added": "81"
```
```suggestion
              "version_added": "84"
```

foolip reviewed (commented):
Updating from collector results revealed that I forgot to mirror Edge data in two places.

Also, much of this is supported on `Element` in Safari and Safari iOS. That requires some digging.

foolip commented on api/_mixins/ARIAMixin__ElementInternals.json:
```diff
@@ -0,0 +1,1830 @@
+              "version_added": "83"
+            },
+            "edge": {
+              "version_added": "81"
```
```suggestion
              "version_added": "83"
```

foolip commented on api/_mixins/ARIAMixin__ElementInternals.json:
```diff
@@ -0,0 +1,1830 @@
+              "version_added": "84"
+            },
+            "edge": {
+              "version_added": "81"
```
```suggestion
              "version_added": "84"
```

foolip reviewed (commented):
Oops there was more.

foolip commented:
OK, I've figured out the story for Safari now, but I'll merge this and send a separate PR.

foolip commented:
I've sent https://github.com/mdn/browser-compat-data/pull/9265 to add the Safari data. @ddbeck if you're going to do a release today, it would be great if that's included as well :)
